src/scripts/create_sst5.py

```diff
@@ -0,0 +1,84 @@
+"""Create the SST5-mini sentiment dataset and upload it to the HF Hub."""
```

Comment:
sst5 vs sst2?

Reply:
SST2 and SST5 is really the same dataset, but SST2 only has two labels (positive and negative). To maintain consistency with the other sentiment datasets I went for "SST3", being SST5 where we merge the "very positive/negative" with "positive/negative".